packages/web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/web-components",
-  "version": "4.15.0",
+  "version": "5.0.0",
```

Comment:
Nothing changed in `web-components`, so let's drop this back down in the next PR.